autonomy/data/Dockerfiles/agent/start.sh

```diff
@@ -30,25 +26,20 @@ else
         aea generate-key ethereum
     fi
 fi
-if [ "$INSTALL" == "1" ];
+
+if [ "$AEA_PASSWORD" != "" ];
 then
-    echo "Installing the necessary dependencies!"
-    aea install && cd .. && aea delete agent
```

Comment:
previously we deleted, now that's not needed I guess because we want to run this exact agent? What if I want to run two agents from the same image, then they have different overrides...

Reply:
The overrides will only be updated at the runtime in containers, the image won't have any effect from running one agent.